### PR TITLE
Added networkx and nose as requirements

### LINKED ISSUE #20: Add networkx and nose as setup dependencies
In `setup.py`, `install_requires` should include `networkx>=2.0`, and `tests_requires` should include `nose` (I am not sure what minimum version of `nose` is needed, but NetworkX uses `nose>=0.10.1`).

### PR DESCRIPTION
**Pull request by OrkoHunter**
Fixes #20

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -61,5 +61,6 @@
         namespace_packages = ['networkx.addons', 'networkx'],
         libraries          = libraries,
         ext_modules        = ext_modules,
-        install_requires   = install_requires
+        install_requires   = install_requires,
+        tests_require      = ['nose>=0.10.1']
```
Comment (ysitu) [inline setup.py]: Also add `test_suite = 'nose.collector'`.

Comment (OrkoHunter) [inline setup.py]: Added `test_suite`.

**Comment by ysitu** [comment]
Can you verify that `python setup.py test` works?

**Comment by OrkoHunter** [comment]
Well it throws out saying
```
Failure: OSError ([Errno 20] Not a directory: '/usr/local/lib/python2.7/dist-packages/networkx-metis-1.0-py2.7-linux-x86_64.egg/networkx') ... ERROR
```
Complete log
```
======================================================================
ERROR: Failure: OSError ([Errno 20] Not a directory: '/usr/local/lib/python2.7/dist-packages/networkx-metis-1.0-py2.7-linux-x86_64.egg/networkx')
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/usr/lib/python2.7/dist-packages/nose/loader.py", line 411, in loadTestsFromName
    addr.filename, addr.module)
  File "/usr/lib/python2.7/dist-packages/nose/importer.py", line 47, in importFromPath
    return self.importFromDir(dir_path, fqname)
  File "/usr/lib/python2.7/dist-packages/nose/importer.py", line 86, in importFromDir
    if (self.sameModule(old, filename)
  File "/usr/lib/python2.7/dist-packages/nose/importer.py", line 131, in sameModule
    if _samefile(mod_path, new_path):
  File "/usr/lib/python2.7/posixpath.py", line 162, in samefile
    s1 = os.stat(f1)
OSError: [Errno 20] Not a directory: '/usr/local/lib/python2.7/dist-packages/networkx-metis-1.0-py2.7-linux-x86_64.egg/networkx'
-------------------- >> begin captured logging << --------------------
root: INFO: Generating grammar tables from /usr/lib/python2.7/lib2to3/Grammar.txt
root: INFO: Generating grammar tables from /usr/lib/python2.7/lib2to3/PatternGrammar.txt
--------------------- >> end captured logging << ---------------------

----------------------------------------------------------------------
Ran 1 test in 0.154s

FAILED (errors=1)
```

**Comment by ysitu** [comment]
Hmm... It works for me. I ran it from the repository directory.

**Comment by OrkoHunter** [comment]
@ysitu Great. I'll try after clearing the mess from my directory. There's a lot of it.

**Comment by OrkoHunter** [comment]
Worked for me too.

**Pull request merged**